modules/services/model/schemas/au.com.shiftyjelly.pocketcasts.models.db.AppDatabase/105.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
```

Comment:
I have ended up with another file after testing. Should there be `106.json`? 

<img width="1685" alt="Screenshot 2025-01-06 at 12 03 57 pm" src="https://github.com/user-attachments/assets/05d48e09-531a-44fb-a2a8-5b83d828799c" />

Reply:
I've pushed the `106.json` and removed the updated on `105.json`. I think the problem was because I had updated the database version in another commit, so I guess it messed up the migration. [a8a03bc](https://github.com/Automattic/pocket-casts-android/pull/3395/commits/a8a03bcf7ab06ffb0c1342f6020702cb0e307608)